scripts/shared/deploy.sh

```diff
@@ -6,13 +6,21 @@ source /usr/share/shflags/shflags
 DEFINE_string 'cluster_settings' '' "Settings file to customize cluster deployments"
 DEFINE_string 'deploytool' 'operator' 'Tool to use for deploying (operator/helm)'
 DEFINE_string 'globalnet' 'false' "Deploy with operlapping CIDRs (set to 'true' to enable)"
+DEFINE_string 'cable_driver' 'default' "Cable driver implementation"
+
 FLAGS "$@" || exit $?
 eval set -- "${FLAGS_ARGV}"
 
 globalnet="${FLAGS_globalnet}"
 deploytool="${FLAGS_deploytool}"
 cluster_settings="${FLAGS_cluster_settings}"
-echo "Running with: globalnet=${globalnet}, deploytool=${deploytool}, cluster_settings=${cluster_settings}"
+cableDriver="${FLAGS_cable_driver}"
+
+if [[ "${cableDriver}" == "defult" ]]; then
```

Comment:
Why not just print the value? It would be simplest

Reply:
It will be simplest, but I think that a statement like 
`Running with: globalnet=false, deploytool=operator, cluster_settings=/go/src/github.com/submariner-io/shipyard/scripts/cluster_settings, cableDriver=` 
might mislead users.
What do you think ?

Reply:
I'm fine with `cable_driver=`.

Reply:
OK, changed the file according the suggestion.